pkgs/top-level/all-packages.nix

```diff
@@ -5709,6 +5709,10 @@ with pkgs;
 
   hyprland-protocols = callPackage ../applications/window-managers/hyprwm/hyprland-protocols { };
 
+  hyprlang = callPackage ../by-name/hy/hyprlang/package.nix {
+    stdenv = gcc13Stdenv;
+  };
+
```

Comment:
```suggestion
```
gcc13 is now the default

Reply:
still there. looks like you removed a different one.

Reply:
Oh yea, my bad. But it's also relevant to the PR.